src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftBuilder.java

```diff
@@ -15,31 +17,28 @@
 import javax.servlet.ServletException;
 import java.io.IOException;
 import java.util.List;
-import java.util.Map;
 
 
-public class OpenShiftBuilder extends OpenShiftBaseStep implements IOpenShiftBuilder {
+public class OpenShiftBuilder extends TimedOpenShiftBaseStep implements IOpenShiftBuilder, ITimedOpenShiftPlugin {
 	
     protected final String bldCfg;
     protected final String commitID;
     protected final String buildName;
     protected final String showBuildLogs;
     protected final String checkForTriggeredDeployments;
-    protected final String waitTime;
     protected final List<NameValuePair> envVars;
     
 
     // Fields in config.jelly must match the parameter names in the "DataBoundConstructor"
     @DataBoundConstructor
     public OpenShiftBuilder(String apiURL, String bldCfg, String namespace, List<NameValuePair> env, String authToken, String verbose, String commitID, String buildName, String showBuildLogs, String checkForTriggeredDeployments, String waitTime) {
-    	super(apiURL, namespace, authToken, verbose);
+    	super(apiURL, namespace, authToken, verbose, waitTime);
```

Comment:
again perhaps you and @oatmealraisin can get some face to face time and coordinate this one ... at first blush I'd prefer we handle most of the wait time changes with his PR